src/passes/Print.cpp

```diff
@@ -55,6 +57,19 @@ static Name printableLocal(Index index, Function* func) {
   return name;
 }
 
+// Wrapper for printing signature names
+struct SigName {
+  Signature sig;
+  SigName(Signature sig) : sig(sig) {}
+};
```

Comment:
I want to decouple how `Signature` is printed for debugging and how it is printed as part of the Print pass, which has to be much uglier because it has to comply with the spec's definition of an identifier.

Reply:
So the `operator<<` for `Signature` is only meant to be used for debugging? If so, making the original data stucture's `operator<<` for debugging and making a separate class for actual printing sounds a little weird to me, but I don't have a strong opinion.

Reply:
I think it makes sense because the non-debug version is only ever useful in Print.cpp, but the debugging printing can be useful anywhere Signatures are used.